include/grpcpp/security/tls_credentials_options.h

```diff
@@ -146,6 +146,41 @@ class TlsServerAuthorizationCheckConfig {
       server_authorization_check_interface_;
 };
 
+/// Configuration for Tls key logging.
```

Comment:
Public API additions will need a gRFC drafted and reviewed at github.com/grpc/proposal.

Reply:
Added link to the relevant GRFC to the PR: https://github.com/grpc/proposal/pull/252